packages/fx-core/src/component/driver/devTool/installDriver.ts

```diff
@@ -154,8 +155,12 @@ export class ToolsInstallDriverImpl {
     outputEnvVarNames?: Map<string, string>
   ): Promise<Map<string, string>> {
     const res = new Map<string, string>();
-    const depsManager = new DepsManager(new EmptyLogger(), new EmptyTelemetry());
-    const funcStatus = await depsManager.ensureDependency(DepsType.FuncCoreTools, true);
+    const funcToolChecker = new FuncToolChecker(new EmptyLogger(), new EmptyTelemetry());
+    const funcStatus = await funcToolChecker.resolve({
+      version: "4",
+      symlinkDir: "./devTools/func",
```

Comment:
Add to gitignore in template?

Reply:
Yes, we will add it to .gitignore and .funcignore